docs/writing-tests/test-runner.md

```diff
@@ -0,0 +1,293 @@
+---
+title: 'Test runner'
+---
+
+Storybook test runner turns all of your stories into executable tests. It is powered by [Jest](https://jestjs.io/) and [Playwright](https://playwright.dev/).
+
+- For those [without a play function](../writing-stories/introduction.md): it verifies whether the story renders without any errors.
+- For those [with a play function](../writing-stories/play-function.md): it also checks for errors in the play function and all assertions passed.
+
+These tests run in a live browser and can be executed via the [command line](#cli-options) or your [CI server](#set-up-ci-to-run-tests).
+
+## Setup
+
+The test runner is a standalone, framework-agnostic addon that runs parallel to your Storybook. You will need to take some additional steps to set it up properly. Detailed below is our recommendation to configure and execute it.
+
+Run the following command to install the addon and the required dependencies.
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'common/storybook-test-runner-install.yarn.js.mdx',
+    'common/storybook-test-runner-install.npm.js.mdx',
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+Update your `package.json` scripts and enable the test runner.
+
+```json
+{
+  "scripts": {
+    "test-storybook": "test-storybook"
+  }
+}
+```
+
+Start your Storybook with:
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'angular/storybook-run-dev.with-builder.js.mdx',
+    'common/storybook-run-dev.npm.js.mdx',
+    'common/storybook-run-dev.yarn.js.mdx',
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+<div class="aside">
+💡 Storybook's test runner requires a running Storybook instance to run all the existing tests.
```

Comment:
Not strictly true. It can also run against a built Storybook: https://github.com/storybookjs/test-runner/#running-against-a-deployed-storybook

Reply:
> 💡 Storybook's test runner requires either a locally running or published Storybook instance  to run all the existing tests.

Reply:
With the latest commits. I've updated based on the conversation I and @kylegach had earlier today. @winkerVSbecks let me know if you're ok with it as well.